travis/travis-build.sh

```diff
@@ -97,6 +74,11 @@ linuxdeploy/AppRun --list-plugins
 export UPDATE_INFORMATION="gh-releases-zsync|AppImage|appimaged|continuous|appimaged*$ARCH*.AppImage.zsync"
 export SIGN=1
 export VERBOSE=1
-linuxdeploy/AppRun --appdir appdir --output appimage
+# Add "hidden" dependencies; https://github.com/AppImage/libappimage/issues/104
+linuxdeploy/AppRun \
+    --appdir appdir \
+    --library=/usr/lib/x86_64-linux-gnu/librsvg-2.so.2 \
```

Comment:
We need to make librsvg and libcairo hard dependencies of libappimage so that this kind of extra step won't be required. Otherwise some users will end up with a broken experience, which we should avoid.